invoicer_project/static/js/login_request.js

```diff
@@ -0,0 +1,105 @@
+const host = "http://127.0.0.1:8000";
+const emailForm = document.getElementById("email_input_log_in_page");
+const passwordForm = document.getElementById("password_input_log_in_page");
+
+
+function validatePassword() {
+    return passwordForm.value !== '';
+}
+
+function validateEmail() {
+    let user_email = emailForm.value;
+    return !(user_email.includes(' ') || !(/^[a-zA-Z0-9.]{3,20}@(?:[a-zA-Z0-9]{2,20}\.){1,30}[a-zA-Z]{2,10}$/.test(user_email))) 
+}
+
+function validateLoginDataOnFrontEnd() {
+    const emailRes = validateEmail();
```

Comment:
We should think of changing emailRes to emailInputValidationResult
or
if we want to shorten it let's consider something like emInpValRes

Reply:
Personally I prefer emailInputValidationResult

Reply:
emailInputValRes, done